kpi/views/v2/data.py

```diff
@@ -20,7 +21,7 @@
 
 
 class DataViewSet(AssetNestedObjectViewsetMixin, NestedViewSetMixin,
```

Comment:
@noliveleger, I'm getting ``AssertionError: 'DataViewSet' should either include a `queryset` attribute, or override the `get_queryset()` method.`` if I try to access the data endpoint in a browser without specifying a format, e.g. `http://kf.kobo.local:9000/api/v2/assets/aS8uoRSD5C9qNtkr35GUWd/data/?limit=30&start=0&sort={%22_id%22:-1}`.

Also, if I request XML format and still include `sort={%22_id%22:-1}`, I get `FieldError: Cannot resolve keyword u'_id' into field. Choices are: date_created, date_modified, deleted_at, id, status, user, user_id, uuid, xform, xform_id, xml`.

Reply:
Hmm. Good catch. I always tested it with the API, KPI UI and unittests which all use JSONRenderer by default.

For the `XML` format, I have desactivated the sort. I've forced `sort` by id. 
I'll open an issue to restore sorting for `XML`.